documentation/book/con-product-downloads.adoc

```diff
@@ -5,14 +5,12 @@
 [id='downloads-{context}']
 = {ProductName} Downloads
 
-{ProductName} releases are available for download from {ReleaseDownload}.
 ifdef::Downloading[]
-The release artifacts contain documentation
-and example YAML files for deployment on {ProductPlatformName}.
-The example files are used throughout this documentation and can be used to install {ProductName}.
-There is a Helm Chart for deployment the Cluster Operator, using link:https://helm.sh/[Helm^], as well.
+{ProductName} releases are available to download from {ReleaseDownload}. The release artefacts contain documentation and example `.yaml` files for deployment on {ProductPlatformName}. The example files are used throughout this documentation and can be used to install {ProductName}. Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
 endif::Downloading[]
 ifndef::Downloading[]
-{ExampleYAMLURL} for deployment on {ProductPlatformName} are used throughout this documentation.
+The following procedures use example `.yaml` files to help you install {ProductName} on {ProductPlatformName}, perform common operations, and configure your Kafka cluster. To access these `.yaml` files, download and extract the `examples.zip` file from http://amq.io/amqstreams/
```

Comment:
* Wouldn't using just "YAML" instead of `.yaml` be better?
* I think all other links are in the style link:http://amq.io/amqstreams/[AMQ Streams website^]. I also think you should use here a variable (such as {ReleaseDownload}), which will make it easier to fix this address if it changes.

Reply:
Good spot. I'll change to YAML because we're referring to the file type rather than the file name.
Yep I'll use a variable.